validator/client/src/test/java/tech/pegasys/teku/validator/client/loader/ExternalValidatorSourceTest.java

```diff
@@ -116,4 +128,83 @@ void shouldThrowExceptionWhenDeleteValidator() {
   void shouldSayFalseToAddValidators() {
     assertThat(validatorSource.canUpdateValidators()).isFalse();
   }
+
+  @Test
+  void shouldAddValidator_getsConfigUrlWhenNotProvided(@TempDir Path tempDir) throws IOException {
```

Comment:
name more like `addValidator_shouldGetConfigUrlWhenNotProvided`
function under test, then underscore, then shouldXXXX where XXXX is desired behaviour